packages/atomic-react/src/components/index.ts

```diff
@@ -1,6 +1,8 @@
 export * from './stencil-generated/index';
 export * from '@coveo/atomic/headless';
 
-// Important: Re-exporting under the same name "AtomicSearchInterface" shadows the original component
+// Important: Re-exporting under the same name (eg: "AtomicSearchInterface") shadows the original component
 // and should wrap it nicely for users of the library
 export {AtomicSearchInterfaceWrapper as AtomicSearchInterface} from './SearchInterfaceWrapper';
+export {ResultTemplateWrapper as AtomicResultTemplate} from './ResultTemplateWrapper';
+export {TemplateWrapper as Template} from './TemplateWrapper';
```

Comment:
Could we combine these two components into one?

Reply:
Yes we could have only the `ResultTemplateWrapper` automatically output the template tag with inner HTML.

I think that's a great idea.